jenkins/README.md

```diff
@@ -26,11 +26,11 @@ publicly visible will get admin rights on the CI jobs. This means :
 
 We have multiple jobs that run some integration tests. The jobs can be
 triggered on PR from metal3-dev-env, baremetal-operator and
-cluster-api-provider-baremetal repositories by commenting the commands below.
+cluster-api-provider-metal3 repositories by commenting the commands below.
 The job result will be posted as a comment.
 
- * **/test-integration** run integration tests for V1alpha1 on Ubuntu
- * **/test-centos-integration** run integration tests for V1alpha1 on CentOS
+ * **/test-integration** run integration tests for master on Ubuntu
```

Comment:
shouldn't it be "run v1alpha3 integration tests for master on *..."

Reply:
As we discussed offline, we'll change the jobs after the release on 10 th of March, so this will be correct then